lib/pusher/client.rb

```diff
@@ -269,13 +269,35 @@ def trigger(channels, event_name, data, params = {})
       post('/events', trigger_params(channels, event_name, data, params))
     end
 
+    # Trigger multiple events at the same time
+    #
+    # POST /apps/[app_id]/batch_events
+    #
+    # @param events [Array] List of events to publish
```

Comment:
i think it's good enough, i don't know if `@param` supports variadic arguments. either case it works the same so it's not a biggie if the user misses it